Sources/UILabelExtensions.swift

```diff
@@ -65,6 +65,19 @@ extension UILabel {
             self.text = _text
         }, completion: nil)
     }
+   
+    // Set lineSpacing for UILabel
+    public func setLineSpacing(lineSpacing: CGFloat) {
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.lineSpacing = 1.0
+        paragraphStyle.lineHeightMultiple = lineSpacing
```

Comment:
This is confusing. Does this mean your spacing = 1.0 * lineHeight = lineSpacing. If so, you could just call the method arg as lineHeightMultiple.